packages/E2ETest/wdio/pages/BasePage.ts

```diff
@@ -3,89 +3,44 @@
  * Licensed under the MIT License.
  */
 
-import {
-  REACT_CONTROL_ERROR_TEST_ID,
-  BACK_BUTTON,
-  TREE_DUMP_RESULT,
-} from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
+import { TREE_DUMP_RESULT } from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
+
+const ELEMENT_LOADED_TIMEOUT = 60000;
 
 export function By(testId: string): WebdriverIO.Element {
   return $('~' + testId);
 }
 
-export function wait(timeout: number) {
-  return new Promise(resolve => {
-    setTimeout(resolve, timeout);
-  });
-}
-
 export class BasePage {
   isElementLoaded(element: string): boolean {
     return By(element).isDisplayed();
   }
 
-  waitForElementLoaded(element: string, timeout?: number) {
+  waitForElementLoaded(element: string) {
     browser.waitUntil(
-      () => {
-        return this.isElementLoadedOrLoadBundleError(element);
-      },
-      this.timeoutForPageLoaded(timeout),
-      'Wait for element ' + element + ' timeout'
+      () => this.isElementLoaded(element),
+      ELEMENT_LOADED_TIMEOUT,
+      `Failed to find element with testId "${element}" within ${ELEMENT_LOADED_TIMEOUT}ms`
     );
   }
 
-  getTreeDumpResult(): boolean {
-    var testResult = false;
-    const maxWait = 20;
-    var waitCount = 1;
-    do {
```

Comment:
The timeout logic here was pretty busted up. Because the wait is faulty (the bug on that was closed), we just try to look for a tree dump 20 times without a wait. Fixed this to just use the polling logic the rest of the world uses.